File: SchoolRegisterApp.Models/Dtos/SchoolDto.cs
﻿using SchoolRegisterApp.Models.Entities;
using SchoolRegisterApp.Models.Enums;

namespace SchoolRegisterApp.Models.Dtos
{
    public class SchoolDto
    {
        public int Id { get; set; }

        public string Name { get; set; }

        public string NameAlt { get; set; }

        public SchoolTypeEnum Type { get; set; }

        public string Settlement { get; set; }

        //public Settlement Settlement { get; set; }

        public bool IsActive { get; set; }
    }
}

File: school-register-app/src/school/dtos/school-dto.ts
import { SchoolTypeEnum } from "../../enums/school-type.enum";
import { SettlementDto } from "../../settlement/dtos/settlement-dto";

export class SchoolDto{
  id: number;
  name: string;
  nameAlt: string;
  type: SchoolTypeEnum;
  settlement: SettlementDto;
  isActive: boolean
}